internal/advtools/fastcombat.lua

```diff
@@ -0,0 +1,91 @@
+--@ module=true
+local overlay = require('plugins.overlay')
+
+-- Overlay
+AdvCombatOverlay = defclass(AdvCombatOverlay, overlay.OverlayWidget)
+AdvCombatOverlay.ATTRS{
+    desc='Faster combat!',
+    default_enabled=true,
+    viewscreens='dungeonmode',
+    frame={w=0, h=0},
+}
+local skip_combat = false
+function AdvCombatOverlay:render()
+    local adv = df.global.adventure
+    if adv.player_control_state == df.adventurest.T_player_control_state.TAKING_INPUT then
+        skip_combat = false
+        return
+    end
+    if skip_combat then
+        -- Instantly process the projectile travelling
+        adv.projsubloop_visible_projectile = false
+        -- Skip the combat swing animations
+        adv.game_loop_animation_timer_start = adv.game_loop_animation_timer_start + 1000
+    end
+end
+
+
+COMBAT_MOVE_KEYS={
```

Comment:
please use the standard formatting for tables:
```lua
local TABLE_NAME = {
    var1=value,
    var2=value,
}
```

Alternately, you can use `utils.invert`:
```lua
local COMBAT_MOVE_KEYS = utils.invert{
    '_MOUSE_L', 'SELECT', 'A_MOVE_N', ...
}
```